packages/snaps-execution-environments/src/common/validation.ts

```diff
@@ -180,6 +181,32 @@ export function assertIsOnNameLookupRequestArguments(
   );
 }
 
+export const OnUserInputArgumentsStruct = object({
+  id: string(),
```

Comment:
What is the ID here?

Reply:
That's the id of the interface the event is coming from, that's a spec from the SIP

Reply:
Ah, of course 👍